### PR TITLE
Add unit test of StandaloneContextManagerBuilderText

### PR DESCRIPTION
**Pull request by fwhdzh**
For #12407 .

Changes proposed in this pull request:
- Add unit test of StandaloneContextManagerBuilderText.

### CODE REVIEW

**Comment by strongduanmu** [comment]
@fwhdzh Can you fix ci error first?

**Comment by fwhdzh** [comment]
> @fwhdzh Can you fix ci error first?

Ok, I will fix it soon.

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandaloneContextManagerBuilderTextTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    public static final String TEST_DATA_SOURCE_INNER_MAP = "TEST_DATA_SOURCE_INNER_MAP";
+
+    public static final String TEST_CONNECTION_URL = "jdbc:mysql://testhost:3306/testdatabase";
+
+    @Test
+    public void assertBuild() throws SQLException {
+        Map<String, Map<String, DataSource>> dataSourceMap = getDataSourceMap();
+        Map<String, Collection<RuleConfiguration>> schemaRuleConfigs = getSchemaRuleConfigs();
+        Collection<RuleConfiguration> globalRuleConfigurationCollection = getGlobalRuleConfigurationCollection();
+        final Properties props = new Properties();
```
Comment (strongduanmu) [inline shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandaloneContextManagerBuilderTextTest.java:56]: @fwhdzh Please remove this useless final modifier.

Comment (strongduanmu) [inline shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandaloneContextManagerBuilderTextTest.java:56]: @fwhdzh Please remove this useless final modifier.

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandaloneContextManagerBuilderTextTest.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        Map<String, Collection<RuleConfiguration>> result = new HashMap<>();
+        result.put(TEST_DATA_SOURCE_INNER_MAP, ruleConfigurationCollection);
+        return result;
+    }
+
+    private Collection<RuleConfiguration> getGlobalRuleConfigurationCollection() {
+        RuleConfiguration globalRuleConfiguration = mock(RuleConfiguration.class);
+        Collection<RuleConfiguration> globalRuleConfigurationCollection = new HashSet<>();
+        globalRuleConfigurationCollection.add(globalRuleConfiguration);
+        return globalRuleConfigurationCollection;
```
Comment (strongduanmu) [inline shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandaloneContextManagerBuilderTextTest.java:97]: @fwhdzh Maybe `result` is better.

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/fixture/StandalonePersistRepositoryFixture.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    @Override
+    public void persist(final String key, final String value) {
+        persistMap.put(key, value);
+    }
+
+    @Override
+    public void delete(final String key) {
+
```
Comment (strongduanmu) [inline shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/fixture/StandalonePersistRepositoryFixture.java:57]: @fwhdzh Please remove useless blank line.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

**Review by strongduanmu — APPROVED** [review]

**Pull request merged**